### DISCUSSION TITLE
Translation ?

### DISCUSSION THREAD

**Discussion by TheCamel** [MahApps/MahApps.Metro#4425] [Q&A] [answered]
is it possible to change the label in the dialog ? or should i use custom ?
because ok/cancel don't always fit :-)

**Comment by timunie** [comment, marked as answer]
https://github.com/MahApps/MahApps.Metro/blob/221996ba181414de311c7550a774d83555631258/src/MahApps.Metro.Samples/MahApps.Metro.Demo/MainWindow.xaml.cs#L240-L247

**Reply by TheCamel** [reply]
Thanks a lot !!
